<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="OptimizeButton">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>600</y>
      <width>300</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>300</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>300</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string>Оптимизировать</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>60</y>
      <width>300</width>
      <height>241</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>300</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>300</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="title">
     <string>Параметры функции</string>
    </property>
    <widget class="QWidget" name="layoutWidget_3">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>33</y>
       <width>261</width>
       <height>26</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Точка старта</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="StartField">
        <property name="text">
         <string>3 3</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_4">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>65</y>
       <width>266</width>
       <height>26</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Коордитаны коробки</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="AreaField">
        <property name="text">
         <string>-5 5 -5 5</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QComboBox" name="FunctionSelect">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>261</width>
       <height>24</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>x^2+y^2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>(1-x)^2+100(y-x^2)^2</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>98</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Функция</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>310</y>
      <width>300</width>
      <height>271</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>300</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>300</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="title">
     <string>Параметры метода</string>
    </property>
    <widget class="QWidget" name="layoutWidget_5">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>60</y>
       <width>138</width>
       <height>52</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="RandomSearch">
        <property name="text">
         <string>Случайный поиск</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="Newton">
        <property name="text">
         <string>Ньютон</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Метод</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>130</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Критерий остановки</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_RS">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>170</y>
       <width>181</width>
       <height>24</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Число итераций</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Итерации улучшение</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="comboBox_N">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>170</y>
       <width>181</width>
       <height>24</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Норма градиента</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Норма разности последних точек</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Норма отношения</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Число итераций</string>
      </property>
     </item>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>220</y>
       <width>265</width>
       <height>26</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Значение параметра</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit">
        <property name="text">
         <string>100000</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>800</width>
      <height>600</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>800</width>
      <height>600</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
